public/diagnostic.js

```diff
@@ -23,3 +23,39 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+const Order = Ember.Object.extend({
+  unitPrice: this.unitPrice,
+  quantity: this.quantity,
+  orderPrice: Ember.computed('this.unitPrice','this.quantity', function(){
+    return (this.unitPrice * this.quantity);
+  }),
+});
+
+let hats = Order.create({
+  unitPrice: 5,
+  quantity: 2,
+  orderPrice
+});
+
+let lamp = Order.create({
+  unitPrice: 20,
+  quantity: 1,
+  orderPrice
+});
+
+let handTowels = Order.create({
+  unitPrice: 8,
+  quantity: 3,
+  orderPrice
+});
```

Comment:
You need to pass a price to these objects, and create a new cart:
```js
let newCart = Cart.create({
  orders: [
  Ember.object.create({name: 'hat', quantity: 2, price: 5}),
  Ember.object.create({name: 'desk lamp', quantity: 1, price: 20}),
  Ember.object.create({name: 'hand towels', quantity: 3, price: 8}),
]
});
```

This is a more compact way to do it, but you could have done it your way and then had
```
let newCart = Cart.create({
 orders: [ hats, lamp, handTowels]
});
```